inspirehep/modules/refextract/tasks.py

```diff
@@ -55,7 +55,7 @@ def create_journal_kb_file():
         FROM
             records_metadata AS r
         WHERE
-            (r.json -> '_collections')::jsonb ? 'Journals'
+            (r.json -> '_collections')::jsonb ? 'Journals' AND ((r.json -> 'deleted') IS NULL OR r.json['deleted'] != 'true'::jsonb)
```

Comment:
isn't `r.json['deleted']` a SQLAlchemy thing? or is it new PostgreSQL syntax? also, isn't `NULL != 'true'::jsonb`, making the first clause redundant?

Reply:
Actually it doesn't work correctly without adding the clause with `NULL`. `r.json['deleted']` also works but I would change it to be consistent with the rest of the query